grafast/grafserv/src/servers/lambda/index.ts

```diff
@@ -0,0 +1,132 @@
+import type {
+  APIGatewayEvent,
+  APIGatewayProxyEvent,
+  APIGatewayProxyResult,
+  Context as LambdaContext,
+} from "aws-lambda";
+import type {
+  GrafservConfig,
+  RequestDigest} from "grafserv";
+import {
+  GrafservBase,
+  processHeaders
+} from "grafserv";
+import { toArray } from "modern-async";
+
+export class LambdaGrafserv extends GrafservBase {
+  createHandler() {
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
+    const serv = this;
+    return async function (
+      event: APIGatewayEvent,
+      context: LambdaContext,
+    ): Promise<APIGatewayProxyResult> {
+      const version = event.requestContext.protocol.match(
+        "^HTTP/(?<major>[0-9]+).(?<minor>[0-9]+)$",
```

Comment:
Let's escape the `.` and use a regex literal rather than casting a string:

```suggestion
        /^HTTP\/(?<major>[0-9]+)\.(?<minor>[0-9]+)$/,
```